tests/puppeteer/pages/BO/international/languages/index.js

```diff
@@ -130,4 +139,59 @@ module.exports = class Languages extends LocalizationBasePage {
     await this.clickAndWaitForNavigation(this.deleteRowLink.replace('%ROW', row));
     return this.getTextContent(this.alertSuccessBlockParagraph);
   }
+
+  /* Bulk Actions Methods */
+  /**
+   * Enable / disable Suppliers by Bulk Actions
+   * @param enable
+   * @return {Promise<textContent>}
+   */
+  async bulkEditEnabledColumn(enable = true) {
```

Comment:
```suggestion
  async bulkEditEnabledColumn(isEnabled = true) {
```

Reply:
This function is not checking status, but changing it. Maybe naming param as `toEnable` is better, WDYT ?

Reply:
Agree with `toEnable`